src/monitor/version_compat.h

```diff
@@ -14,6 +14,11 @@
 
 #include "postgres.h"
 
+/* we do not ssupport PG 12 yet*/
```

Comment:
Typo on “support”.

Also, I think we only support Postgres 10 and 11, so we might as well change the test to check for the supported versions?

Reply:
are you okay with use of #pragma ?

Reply:
done (without pragma)